# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../adm8211/linux-2.6.17.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- adm8211/adm8211_hw.c.vanilla	2006-07-01 17:17:45.000000000 +0000
+++ adm8211/adm8211_hw.c	2006-07-01 17:20:10.000000000 +0000
@@ -39,6 +39,7 @@
 #include <linux/etherdevice.h>
 #include <linux/ioport.h>
 #include <linux/pci.h>
+#include <linux/dma-mapping.h>
 #include <linux/if_arp.h>
 #include <linux/delay.h>
 #include <linux/crc32.h>
